src/NUnitFramework/nunitlite-runner/nunitlite-runner.csproj

```diff
@@ -1,22 +1,6 @@
-﻿<Project>
-  <PropertyGroup Condition="'$(OS)' == 'Windows_NT' and '$(TargetFramework)' != 'netcoreapp2.1'">
-    <UseWindowsFormsAndWPF>true</UseWindowsFormsAndWPF>
```

Comment:
@jnm2 this was causing no tests to be run and the app returning immediately. I think you added it so you could test the WPF and WinForms code, but when I ran at the CLI it returned immediately with no output. Even `nunitlite-runner.exe --help` and `nunitlite-runner.exe --version` didn't output anything. I don't think that was intended, so I've reverted for now. Maybe we need to wrap the tests that you intended to run in NUnitLite?